services/ui-src/src/forms/mcpar/cpc/cpc.schema.ts

```diff
@@ -1,3 +1,22 @@
 import { object } from "yup";
+import {
+  checkbox,
+  date,
+  nested,
+  number,
+  radio,
+  text,
+} from "utils/forms/schemas";
 
-export default object({});
+export default object({
+  "cpc-1": text(),
+  "cpc-1-b": date(),
+  "cpc-2": text(),
```

Comment:
We actually have a validation schema just for urls:
```suggestion
  "cpc-2": url(),
```